scripts/ci/run-test-mac-sdk.sh

```diff
@@ -8,6 +8,7 @@ export PATH=${MONO_REPO_ROOT}/external/bockbuild/stage/bin:$PATH
 # Bundled MSBuild
 cd ${MONO_REPO_ROOT}/external/bockbuild/builds/msbuild-15/
 ${TESTCMD} --label="msbuild-tests" --timeout=180m ./build.sh -host mono -configuration Release
+zip msbuild-test-results.zip artifacts/2/Release-MONO/TestResults/*
```

Comment:
this should use `${MONO_REPO_ROOT}/msbuild-test-results.zip` so it ends up in the repo root

Reply:
Ok. How do I get the same path in `msbuild.py`, to copy the binlogs?

Reply:
I actually don't know, we probably need to bug Alexis about it :)